### ISSUE TITLE
createDynamicShortLink - Unable to create link

### ISSUE THREAD

**Issue by swapnaliDeveloper** [Pantrist-dev/capacitor-firebase-dynamic-links#27]
Plugin version - https://www.npmjs.com/package/@pantrist/capacitor-firebase-dynamic-links/v/4.1.0

```
 const config: LinkConfig = {
              domainUriPrefix: "...",
              uri: uristring,
              androidParameters: {
                packageName:"..."
              },
              iosParameters: {
                bundleId: "...",
                appStoreId:"..."     
              }
```

createDynamicLink => link getting generrated successfully with above config 
but 
createDynamicShortLink => getting error as following in only ios platform 
{errorMessage: "Unable to create link", message: "Unable to create link"}

during Debugging Dynamic Link getting following warning , not getting any documentation related warning , because of this warning ios createDynamicShortLink failing ?

<img width="923" alt="image" src="https://user-images.githubusercontent.com/104262116/224676293-63285b20-c877-4558-a6ff-870fea0b8c35.png">

**Comment by NLueg** [comment]
Hi there. I don't think that it's because of the warning. Maybe the `ipadBundleId` is required?
It's actually hard to tell what the problem could be as the message currently is only generic.
I just published Version `4.1.2` which should print the error message. Maybe this helps to find the real problem.

**Comment by swapnaliDeveloper** [comment]
createDynamicShortLink working with updated version :)
